packages/camera/lib/camera.dart

```diff
@@ -95,27 +102,32 @@ class CameraPreview extends StatelessWidget {
   }
 }
 
+// Metadata of the camera plugin
 class CameraValue {
   /// True if the camera is on.
-  final bool isStarted;
+  final bool opened;
```

Comment:
Perhaps: `isPreviewStarted` ?

Reply:
I kept the *open* to refere to the *openCamera* function (which is also the name Android is using). But I changed all the bool class variables to use the *isSomething* notation. The camera can be opened without the preview being started in Android (the preview is closed when the camera start recording, then started again when the recording is stopped).